hack/make/test-integration-cli

```diff
@@ -2,7 +2,7 @@
 set -e
 
 bundle_test_integration_cli() {
-	TESTFLAGS="$TESTFLAGS -check.v"
+	TESTFLAGS="$TESTFLAGS -check.v -check.timeout=${TIMEOUT} -timeout=360m"
```

Comment:
Does just `-timeout` here work, or does it have to be `-test.timeout` explicitly?